src/hb-ot-cmap-table.hh

```diff
@@ -576,9 +576,13 @@ struct CmapSubtableLongSegmented
 struct CmapSubtableFormat12 : CmapSubtableLongSegmented<CmapSubtableFormat12>
 {
   static hb_codepoint_t group_get_glyph (const CmapSubtableLongGroup &group,
-					 hb_codepoint_t u)
-  { return likely (group.startCharCode <= group.endCharCode) ?
-	   group.glyphID + (u - group.startCharCode) : 0; }
+					 hb_codepoint_t u, unsigned int num_glyphs = UINT_MAX)
+  { hb_codepoint_t g = group.glyphID;
```

Comment:
Feel free to use new line here, we use curly braces in the same for one liner functions.